crypto/rand/rand_unix.c

```diff
@@ -93,6 +93,27 @@ static uint64_t get_timer_bits(void);
 # error "UEFI and VXWorks only support seeding NONE"
 #endif
 
+#if defined(OPENSSL_SYS_VXWORKS) && defined(OPENSSL_RAND_SEED_NONE)
```

Comment:
Please remove ` && defined(OPENSSL_RAND_SEED_NONE)`. This condition is already enforced by lines 91-94.


**Side note**: while pondering whether one couldn't use the occasion to implement a seed source for VxWorks, I found this interesting article [33C3: Analyzing Embedded Operating System Random Number Generators](http://samvartaka.github.io/cryptanalysis/2017/01/03/33c3-embedded-rngs) from January 2017, which mentions OpenSSL and VxWorks and essentially states that hoping for a proper entropy source on VxWorks is futile...

Reply:
ok, removed it.
we are currently thinking about how to deal with the random source, vxworks 7 has some support for it, also we are considered the CAAM unit of the i.mx6 processor we are using...